lib/core-integration/src/Test/Integration/Scenario/API/Transactions.hs

```diff
@@ -1474,6 +1491,91 @@ spec = do
               txs2 <- listTransactions ctx w (Just te) (Just te) Nothing
               length <$> [txs1, txs2] `shouldSatisfy` all (== 0)
 
+    it "TRANS_DELETE_01 - Single Output Transaction" $ \ctx -> do
+        (wa, wb) <- (,) <$> fixtureWallet ctx <*> fixtureWallet ctx
```

Comment:
side note, but we don't need the second wallet to have funds, right? `emptyWallet` would be sufficient here (and save us both a faucet and some time)

Reply:
yes, it is enough for the recepient wallet to be empty -> corrected